css/json.human.css

```diff
@@ -1,27 +1,48 @@
+@import url(http://fonts.googleapis.com/css?family=Inconsolata);
```

Comment:
can you split this into another css file? something like json.human.inconsolata.css? this is to make it easier for users to opt in to the font and make it easier to use their own fonts.